embulk-standards/src/main/java/org/embulk/standards/GzipFileEncoderPlugin.java

```diff
@@ -21,10 +21,9 @@
     public interface PluginTask
             extends Task
     {
-        // TODO java.util.zip.GZIPOutputStream doesn't support compression level
-        //@Config("level")
-        //@ConfigDefault("6")
-        //public int getLevel();
+        @Config("level")
+        @ConfigDefault("-1")
```

Comment:
default level actually means 6. Java's Deflater simply calls deflateInit2(3) and Z_DEFAULT_COMPRESSION is 6 at the current version of zlib. I guess using 6 here is easier for writing validation code and understanding for users.